api/test/docker/manager-api-conf.yaml

```diff
@@ -15,20 +15,24 @@
 # limitations under the License.
 #
 
+
```

Comment:
I do not think that is a good way to copy the `api/conf/conf.yaml` again only for testing.

we should use `cp` + `sed` to generate `manager-api-conf.yaml`